### PR TITLE
Improved FE DX: linting fixes

### PR DESCRIPTION
**Pull request by fhenrich33**
Follow up for https://github.com/boxwise/boxtribute/pull/1460 to fix linting errors on CI FE build step.

### CODE REVIEW

**Comment by jamescrowley** [comment]
@fhenrich33 @pylipp anything I can do to help get this and/or #1460 ready to be merged? :)

**Comment by fhenrich33** [comment]
> @fhenrich33 @pylipp anything I can do to help get this and/or #1460 ready to be merged? :)

@jamescrowley Me and @HaGuesto are going to review and address this soon!

File: front/src/index.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ if (sentryDsn) {
       new CaptureConsole({
         levels: ["error"],
       }),
+      // eslint-disable-next-line import/namespace
```
Comment (HaGuesto) [inline front/src/index.tsx:31]: can we get rid of this?

File: front/src/tests/test-utils.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
-/* eslint-disable react/function-component-definition */
-/* eslint-disable no-console */
+/* eslint-disable import/export */
```
Comment (HaGuesto) [inline front/src/tests/test-utils.tsx:1]: What is complaining here?

Comment (HaGuesto) [reply front/src/tests/test-utils.tsx:1]: ah I see the next line.

File: front/src/views/Boxes/components/Checkbox.tsx [hunk 3]
```diff
@@ -26,6 +24,7 @@ const useCombinedRefs = (...refs): React.MutableRefObject<any> => {
   return targetRef;
 };
 
+// eslint-disable-next-line react/display-name
```
Comment (HaGuesto) [inline front/src/views/Boxes/components/Checkbox.tsx:27]: why is it complaining here?

**Review by HaGuesto — APPROVED** [review]
well done @fhenrich33 ! These kind of PRs are always annoying. In general feel free to merge, but again please explain why there are two new cases in which you ignore eslint messages. I tagged them in the code.

File: front/src/index.tsx (hunk 1 above)
Comment (fhenrich33) [reply front/src/index.tsx:31]: Until I find out why ESLint or TS is not finding the method even though it's clearly exported, we need to leave it there.

File: front/src/views/Boxes/components/Checkbox.tsx (hunk 3 above)
Comment (fhenrich33) [reply front/src/views/Boxes/components/Checkbox.tsx:27]: It's complaining about missing the component display name, I imagine some conflict with `forwardRef`.

**Comment by pylipp** [comment]
[2.0 FE: fix automated linting, formatting and type checking in FE before commiting](https://trello.com/c/gtV5lYon/1428-20-fe-fix-automated-linting-formatting-and-type-checking-in-fe-before-commiting)

**Pull request merged**